CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.40.0
+* Bug - Register custom logger before it gets used to ensure that the formatter is initiated before it is loaded [#2438](https://github.com/aws/amazon-ecs-agent/pull/2438)
+* Enhancement - adds a jitter to this so that we don't query docker for every container's state all at the same time [#2444] (https://github.com/aws/amazon-ecs-agent/pull/2444)
```

Comment:
extra space after `[#2444]` causing link not rendered;

nit: put enhancement item above bug